Adapter/PlentymarketsAdapter/ResponseParser/Product/Variation/VariationResponseParser.php

```diff
@@ -491,7 +491,7 @@ private function getVariationWeight(array $variation)
      */
     private function getStockLimitation(array $variation)
     {
-        if ($variation['stockLimitation']) {
+        if ($variation['stockLimitation'] == 1) {
```

Comment:
Ist der Vergleich auf 1 notwendig?

Reply:
Jo, da 1 in Plenty für die Beschränkung auf Nettowarenbestand entspricht. 0 für keine Beschränkung und 2 für kein Stockmanagement, daher die Überprüfung auf 1.